google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/GoogleCloudStorageModel.scala

```diff
@@ -0,0 +1,389 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.storage
+
+import akka.annotation.InternalApi
+import akka.http.scaladsl.model.ContentType
+
+import scala.collection.immutable.Seq
+
+object GoogleCloudStorageModel {
```

Comment:
Does the wrapping `object` add anything important?
These classes could just live in `storage`.